src/api/routes/map.ts

```diff
@@ -19,25 +17,11 @@ import sequelize from "../database/sequelize";
 
 const listLandmarks = procedure
   .use(authUser())
-  .input(zLatitude)
-  .output(z.array(zLandmark))
-  .query(async ({ input : latitude }) =>
+  .input(z.array(zLatitude))
```

Comment:
Neither this route nor readMapJsonFiles() should need to take latitudes as input. readMapJsonFiles() can just refer to Latitudes in shared/Map.js

Reply:
this comment isn't addressed.